jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java

```diff
@@ -31,10 +35,13 @@ public static Map<String,Object> formErrorCode(List<Map<String,String>> errorLis
             for(Map<String, String> errorDetail :errorList) {
                 Map<String,Object> error = new HashMap<>();
                 error.put(ERROR, errorDetail.get(ERROR));
+                log.error("error detail - {}",errorDetail.get(ERROR));
```

Comment:
Please be consistent with logging messages.

Update error msg to
"Error details : {}"

space between ,errorDetail.

Reply:
These logger statements are temporarily included to debug qualys data alert issue. They will be reverted once the issue is resolved. In qualys, we are getting data alert but when we check logs, we are not finding any error/exception which caused this data alert. To know what error caused this, we have included the above logger statements.

Reply:
reformatted.